drizzlepac/haputils/deconvolve_utils.py

```diff
@@ -66,10 +68,6 @@ def fft_deconv_img(img, psf, freq_limit=0.95, sigma=None):
         This value should result in selecting a frequency
         one to two orders of magnitude below the largest spectral component of the point-spread function.
 
```

Comment:
Add documentation for the block_size parameter.

Reply:
Fixed in d7c635f